wremnants/theory_tools.py

```diff
@@ -408,14 +408,14 @@ def replace_by_neighbors(vals, replace):
     return vals[tuple(indices)]
 
 def moments_to_angular_coeffs(hist_moments_scales, cutoff=1e-5, sumW2=False):
-    sumW2 = sumW2 and hist_moments_scales._storage_type == hist.storage.Weight
+    hasSumW2 = sumW2 or hist_moments_scales._storage_type == hist.storage.Weight
```

Comment:
Not sure I understand the use of 'or' here. I think the idea was to allow someone to skip the sumW2 if they want, even if the hist already exists.

Reply:
It depends on what we need to do. My impression is that when the input histograms have weight storage type we must set sumW2 to True, or the code will crash afterwards (I think mainly because of some inconsistencies due to sometimes using the flag and sometimes checking the storage type explicitly). 
At that point the argument sumW2 becomes useless, but I don't know if there was a use case to set it explicitly from outside of the function.

Reply:
Well the code should be fixed to not fail when sumw2=False. My understanding is that this argument is meant to drop the sumw2 if it exists and avoid the time spent calculating it